### DISCUSSION TITLE
Problem of accessiblity to Vsual Studio

### DISCUSSION THREAD

**Discussion by Jones-Sw** [community/community#130388] [Accessibility]
### Select Topic Area

General

### GitHub Feature Area

Visual Studio

### Body

There is a problem with Update of the Version 2022 Vs .After trying to download the latest Update of Maui a problem has apeared 
![Uploading Screenshot 2024-06-29 130210.png…]()

**Comment by queenofcorgis** [comment]
Hi @Jones-Sw , we're happy you're here! You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction as your screenshot isn't visible.
